TRACK_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for this PR! :tada:
+
+When we review your submission we will check the following:
+
+- [ ] TODO: Update this list based on the appropriate checks for this specific track
+- [ ] No trailing whitespace nor extraneous blank lines
+- [ ] Proper indentation and consistent code formatting
+- [ ] Free from spelling and grammatical errors
+- [ ] Readable code style
+- [ ] Exemplifies the idioms of {{LANGUAGE}}
+- [ ] Absence of non-standard libraries
+- [ ] No extraneous configuration files
+- [ ] Free of development artifacts such as `puts`, `console.log`, or `debugger` calls
```

Comment:
I like that suggestion, @kotp. I'm going to go ahead and make the locally and merge it.